ibm/service/cdtektonpipeline/resource_ibm_cd_tekton_pipeline.go

```diff
@@ -170,10 +170,15 @@ func ResourceIBMCdTektonPipeline() *schema.Resource {
 								},
 							},
 						},
+						"href": &schema.Schema{
```

Comment:
href is required inside the computed attributes "properties" "definitions"?

Can we fix this across all resources?

Reply:
Fix it so that `href` is marked as "computed" instead of "required" you mean?

Reply:
yes

Reply:
Addressed in the latest commit: https://github.com/IBM-Cloud/terraform-provider-ibm/pull/4235/commits/cbbbe65a29246832e3d02675d4a82db6ac52f6bb